packages/community-cli-plugin/src/commands/start/runServer.js

```diff
@@ -66,7 +66,7 @@ async function runServer(
   const hostname = args.host?.length ? args.host : 'localhost';
   const {
     projectRoot,
-    server: {port},
+    server: {port = 8081},
```

Comment:
Is the `= 8081` here necessary? `metroConfig` should be fully hydrated at this point and populated with the upstream Metro defaults from `metro-config`.

Reply:
I don't think so, but I don't have the context as to why the coalescing was added in the first place so I wanted to make sure we had a failsafe. Happy to remove it if you don't think it's necessary. the less the better.

Reply:
Ah, that might be because Metro's default [used to be](https://github.com/facebook/metro/commit/891fe9674cb6bc63aea4cea365f78fa2584ab138) `8080` whereas RN's was `8081` - CC @huntie who might recall the context there.

Now that Metro and RN are aligned you're right IMO - the less repetition of this value, the better.

Reply:
updated!

Reply:
it seems [something failed](https://github.com/facebook/react-native/pull/44957/checks?check_run_id=26248806780), but I don't have permissions to see what it is. could you let me know?